pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+skip-string-normalization = true
+line-length = 120
+py36 = true
```

Comment:
do we really need this as well as `setup.cfg`?

Reply:
Seems like it

* https://github.com/ambv/black/issues/65
* https://black.readthedocs.io/en/stable/pyproject_toml.html#what-on-earth-is-a-pyproject-toml-file

Reply:
Please remove this and just use `make format`

Reply:
If/when we move to ` pyproject.toml` that's a separate PR and should be done fully, not just for one tool.

Reply:
ok

Reply:
#314